packages/psss/src/api/queries/useSingleWeeklyReport.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { queryCache } from 'react-query';
+import { useTableData } from './useTableData';
+
+const getAlerts = row =>
+  Object.entries(row).reduce(
+    (list, [key, value]) =>
+      key.includes('Threshold Crossed') && value
+        ? [...list, key.split('Threshold Crossed')[0]]
+        : list,
+    [],
+  );
+
+const getUnVerifiedAlerts = (alerts, verifiedStatuses) =>
+  alerts.reduce(
+    (list, syndrome) => (verifiedStatuses.includes(syndrome.id) ? list : [...list, syndrome.id]),
+    [],
+  );
+
+const getEmptyTableData = () => [
+  { id: 'afr', title: 'Acute Fever and Rash (AFR)', percentageChange: 0, totalCases: 0 },
+  { id: 'dia', title: 'Diarrhoea (DIA)', percentageChange: 0, totalCases: 0 },
+  { id: 'ili', title: 'Influenza-like Illness (ILI)', percentageChange: 0, totalCases: 0 },
+  { id: 'pf', title: 'Prolonged Fever (PF)', percentageChange: 0, totalCases: 0 },
+  { id: 'dli', title: 'Dengue-like Illness (DLI)', percentageChange: 0, totalCases: 0 },
+];
+
+const toCommaList = values =>
+  values
+    .join(', ')
+    .toUpperCase()
+    .replace(/,(?!.*,)/gim, ' and');
+
+const getTableData = data => [
```

Comment:
This doesn't feel very maintainable but given the structure of the data I am unsure if there is a better way. I initially wrote this to iterate over the data and parse it but it was just as much code and hardcoded keys and much much more difficult to understand what was going on.